generate/src/Write/Utils.hs

```diff
@@ -34,7 +34,9 @@ docComment docSymbol c = case comment c of
                  _ -> error "'comment' didn't return a comment"
 
 postdocComment :: String -> String
-postdocComment = docComment '^'
+postdocComment c = case docComment '^' c of
+  "" -> ""
+  s -> " " ++ s
```

Comment:
Also, I wish I'd done a PhD, that this could perhaps be a postdoc comment on `postdocComment`